### PR TITLE
#87 #88 F001 and F002 checks

### PR DESCRIPTION
**Pull request by dmius**

### CODE REVIEW

File: resources/checks/F001_heap_bloat.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+sql=$(curl -s -L https://github.com/NikolayS/postgres_dba/raw/master/sql/b1_table_estimation.sql | sed -r ':a; s%(.*)/\*.*\*/%\1%; ta; /\/\*/ !b; N; ba' | sed '/^--/d' | sed '/^$/d')
+sql="$sql"
```
Comment (NikolayS) [inline resources/checks/F001_heap_bloat.sh]: why is this line needed?

File: resources/checks/F001_heap_bloat.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+sql=$(curl -s -L https://github.com/NikolayS/postgres_dba/raw/master/sql/b1_table_estimation.sql | sed -r ':a; s%(.*)/\*.*\*/%\1%; ta; /\/\*/ !b; N; ba' | sed '/^--/d' | sed '/^$/d')
```
Comment (NikolayS) [inline resources/checks/F001_heap_bloat.sh]: Let's used tagged version: https://raw.githubusercontent.com/NikolayS/postgres_dba/4.0/sql/b1_table_estimation.sql

File: resources/checks/F001_heap_bloat.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+sql="$sql"
+sql=${sql%;} #remove last ;
+
+ssh ${HOST} "psql -U postila_ru -t -f - " <<SQL
```
Comment (NikolayS) [inline resources/checks/F001_heap_bloat.sh]: hard-coded username, please fix it

File: resources/checks/F002_index_bloat.sh [hunk 4]
```diff
@@ -0,0 +1,13 @@
+sql=$(curl -s -L https://github.com/NikolayS/postgres_dba/raw/master/sql/b2_btree_estimation.sql | sed -r ':a; s%(.*)/\*.*\*/%\1%; ta; /\/\*/ !b; N; ba' | sed '/^--/d' | sed '/^$/d')
```
Comment (NikolayS) [inline resources/checks/F002_index_bloat.sh]: tagged version: https://raw.githubusercontent.com/NikolayS/postgres_dba/4.0/sql/b2_btree_estimation.sql

**Comment by dmius** [comment]
@NikolayS  @Nastradamus  Ready.

**Comment by NikolayS** [comment]
Thanks! Looks good to me.
